ipatests/test_ipalib/test_config.py

```diff
@@ -563,7 +563,7 @@ def test_finalize_core(self):
         # Test using DEFAULT_CONFIG:
         defaults = dict(constants.DEFAULT_CONFIG)
         (o, home) = self.finalize_core(None, **defaults)
-        assert list(o) == sorted(defaults)
+        assert set(o).issuperset(set(defaults))
```

Comment:
I would rather keep the original check, but make sure that `o` does not contain `fips_mode` before doing it.